src/test/calculate.test.js

```diff
@@ -0,0 +1,9 @@
+import calculate from '../logic/calculate';
+
+const test = { total: '400', next: '200', operation: '+' };
+describe('operate test', () => {
+  it('400 + 200 = 600', () => {
+    const { total } = calculate(test, '=');
+    expect(total).toStrictEqual('600');
+  });
+});
```

Comment:
- [ ] I can see you are testing only one functionality of your calculator logic so please make sure that all the functionalities and scenarios of the `operate.js` and `calculate.js` are tested using jest. :pray:

Reply:
well resolved! ✔️